tests/main/test_predictions_wrapper.py

```diff
@@ -37,23 +37,35 @@ def verify_pickle_serialization(self, model, model_wrapper, test_data):
 
 
 class TestPredictionsWrapperClassification(TestPredictionsWrapper):
-    @pytest.mark.parametrize('dataset_name', ['iris', 'titanic'])
-    def test_prediction_wrapper_classification(self, iris, titanic_simple, dataset_name):
+    @pytest.mark.parametrize('dataset_name', ['iris', 'titanic', 'cancer', 'wine', 'multiclass'])
+    def test_prediction_wrapper_classification(
+            self, iris, titanic_simple, cancer, cancer_booleans, wine,
+            multiclass_classification, dataset_name):
         if dataset_name == 'iris':
```

Comment:
instead of many if/else statements here, consider making a small dictionary and just indexing into it (the python equivalent of a switch in most other languages)

Reply:
Done